### ISSUE TITLE
When will QuickJS have a new release version?

### ISSUE THREAD

**Issue by buddyliao** [bellard/quickjs#355]
It seems that 84 newly commits have been updated since the last release, when will QuickJS have a new release version? Thanks

**Comment by agebhar1** [comment]
The latest release 2024-01-13 is not compatible with the QuickJS engine enabled [NGINX JavaScript version 0.8.6](https://github.com/nginx/njs/commit/c5a29a7af8894ee1ec44ebda71ef0ea1f2a31af6). Appreciate a new release.

**Comment by saghul** [comment]
This might be of interest to you: https://github.com/quickjs-ng/quickjs

**Comment by ctn-malone** [comment]
> This might be of interest to you: https://github.com/quickjs-ng/quickjs

Even if https://github.com/quickjs-ng/quickjs is more active, why not also make a new release here ?

**Comment by saghul** [comment]
I don't have control over this repo.

**Comment by ctn-malone** [comment]
> I don't have control over this repo.

Yes, I know @saghul, sorry 😅. Maybe @chqrlie or @bellard could give us more informations ?

**Comment by chqrlie** [comment]
Hello!
Sorry for the lag, I have been tied up with other commitments lately. Will work on a new release including some of the more recents patches from ng.
Thank you for your interest in QuickJS.
Chqrlie

**Comment by ctn-malone** [comment]
> Hello! Sorry for the lag, I have been tied up with other commitments lately. Will work on a new release including some of the more recents patches from ng. Thank you for your interest in QuickJS. Chqrlie

@chqrlie did you find some time to work on a new release ?

**Comment by ctn-malone** [comment]
Hi @chqrlie, do you think a new QuickJS release can be made to celebrate 2025 ? 🎉

**Comment by chqrlie** [comment]
Yes, but not until I return from CES.
Happy New Year to everyone!

Chqrlie

**Comment by ctn-malone** [comment]
> Hi @chqrlie, do you think a new QuickJS release can be made to celebrate 2025 ? 🎉

Thanks @chqrlie !

**Comment by ctn-malone** [comment]
Any update @chqrlie ?

**Comment by louis123562** [comment]
@chqrlie please provide us a new release :)

**Comment by Sytten** [comment]
Since @bellard is now active again, it might make sense to re-consider merging the projects. There has been a lot of good fixes in quickjs-ng and it seems counter productive to me to keep the two projects separate.

**Comment by bellard** [comment]
A new version was released.

**Comment by Sytten** [comment]
This is good news for quickjs! I do want to re-iterate that it would make more sense to merge the engines, but I get that OSS projects are hard to manage with multiple people.

**Comment by ctn-malone** [comment]
> A new version was released.

Thanks a lot @bellard
